Formula/tweak.rb

```diff
@@ -0,0 +1,15 @@
+lass Tweak < Formula
+  desc "Command-line, ncurses library based hex editor"
+  homepage "https://www.chiark.greenend.org.uk/~sgtatham/tweak/"
+  url "https://www.chiark.greenend.org.uk/~sgtatham/tweak/tweak-3.02.tar.gz"
+  sha256 "5b4c19b1bf8734d1623e723644b8da58150b882efa9f23bbe797c3922f295a1a"
+
+  def install
+    system "make"
+    system "make", "install", "PREFIX=#{prefix}", "MANDIR=#{man}"
```

Comment:
I think this should be:

```suggestion
    system "make", "install", "PREFIX=#{prefix}", "MANDIR=#{man1}"
```